common/persistence/data_manager_interfaces.go

```diff
@@ -348,6 +348,7 @@ type (
 		ClusterReplicationLevel       map[string]int64                  `json:"cluster_replication_level"`
 		DomainNotificationVersion     int64                             `json:"domain_notification_version"`
 		PendingFailoverMarkers        []*types.FailoverMarkerAttributes `json:"pending_failover_markers"`
+		QueueStates                   map[int32]*types.QueueState       `json:"queue_states"`
```

Comment:
Not specific to this place but in general let's add `// Deprecated: ..` comment to the fields/methods that we plan to deprecate. If migration takes a while this would help with readability and avoid confusions

Reply:
I'm planning to add the comment in a later version. Right now, the new queue framework is not introduced yet. I think it's a bit too early to mark those fields deprecated.